src/datasets/bigquery/exact_mau/reference.md

```diff
@@ -0,0 +1,215 @@
+# Exact MAU Data
+
+## Introduction
+
+This article introduces the usage of and methodology behing the "exact MAU"
+tables in BigQuery:
+
+- `firefox_desktop_exact_mau28_by_dimensions`,
+- `firefox_nondesktop_exact_mau28_by_dimensions`, and
+- `firefox_accounts_exact_mau28_by_dimensions`.
+
+The calculation of MAU (monthly active users) has historically been frought
+with troubling details around exact definitions and computational limitations,
+leading to disagreements between analyses.
+These tables contain precalculated MAU, WAU, and DAU aggregates for 
+various usage criteria and dimensions, allowing efficient calculation of
+aggregates across arbitrary slices of those dimensions.
+The tables follow a consistent methodology which is intended as a standard
+across Mozilla for MAU analysis going forward.
+
+## Table of Contents
+
+<!-- toc -->
+
+# Conceptual Model
+
+## Metric
+
+A metric is anything want to (and can) measure.  In order for a metric to be 
+calculated, a _usage criterion_ and a _slice_ must be specified. The metric 
+will produce a single value per day, summarizing data:
+
+- for one day or more days (i.e. the metric value for a particular day may depend on data from other days as well)
+- for all users (whatever notion of user makes sense for the data, generally profiles) in a particular sub-population
+- where the sub-population will include users that meet the specified usage criteria and are in the specified slice.
+
+A simple _usage criterion_ is "All Desktop Activity", which includes all Firefox Desktop users that we have any data (telemetry ping) for on the day in question.  The simplest slice is "All" which places no restrictions on the sub-population.
+
+For example, the metric "Daily Active Users (DAU)" with usage criteria "All Desktop Activity" and slice "All" involves summarizing data on all users of Firefox Desktop over a single day.
+
+## Usage Criteria
+
+A _usage criterion_ is any product or feature for which we wish to calculate
+metrics.  It may be something simple like "All Desktop Activity" (as above) or, 
+similarly, "All Mobile Activity".  It may also be something more specific like
+"Desktop 5 URI Active" corresponding to calculation of 
```

Comment:
`"All Mobile Activity"` and `"All Desktop Activity"` sound like natural language, but this one doesn't. How about this?
```suggestion
"Desktop Total Daily URI >= 5" corresponding to calculation of 
```

Reply:
Or to be consistent with the column naming:
```suggestion
"Desktop Visited 5 URI" corresponding to calculation of 
```

Reply:
Or to use the terminology in active_dau.md
```suggestion
"Active Users Desktop Activity" corresponding to calculation of 
```